lightning/restore/restore.go

```diff
@@ -1166,8 +1184,9 @@ func (t *TableRestore) importEngine(
 }
 
 func (t *TableRestore) postProcess(ctx context.Context, rc *RestoreController, cp *TableCheckpoint) error {
-	if !rc.backend.ShouldPostProcess() {
-		t.logger.Debug("skip post-processing, not supported by backend")
+	// if post-process is disabled or the table is empty, just skip
+	if !rc.backend.ShouldPostProcess() || len(cp.Engines) == 1 {
```

Comment:
should it be `len() > 0` here

Reply:
`len(cp.Engines) == 1` means there is only one opened engine -- the index engine, thus there are no kv in this engine, so it's safe to just skip post process.

Reply:
okay.